src/browser/urls.py

```diff
@@ -101,6 +105,8 @@
         'browser.views.table_clone', name='browser-table_clone'),
 
     # Export
+    url(r'^export/(\w+)/(\w+)/table/(\w+)/get-csv-json/?$', 'browser.views.table_export_csv',
```

Comment:
You can already export tables as CSVs via the query API, https://datahub.csail.mit.edu/api-docs/#!/query/Query_POST. It takes a "select *" and Accept header "text/csv".